packages/cli/src/webhooks/test-webhooks.ts

```diff
@@ -171,7 +171,31 @@ export class TestWebhooks implements IWebhookManager {
 
 			this.clearTimeout(key);
 
-			await this.deactivateWebhooks(workflow);
+			// Check if this is a ChatTrigger webhook - if so, don't deactivate immediately
+			// ChatTrigger needs persistent webhooks for the chat session
+			const isChatTrigger = Object.values(workflow.nodes).some(
+				(node: any) =>
+					node.type === 'n8n-nodes-langchain.chatTrigger' || node.type.includes('chatTrigger'),
+			);
+
+			if (!isChatTrigger) {
```

Comment:
Is this enough? Could a workflow have both webhook and chat trigger nodes, but the webhook gets triggered instead of chat trigger?